src/Faithlife.Tracing.Data/TracingDbConnection.cs

```diff
@@ -112,7 +112,7 @@ protected override void Dispose(bool disposing)
 
 		private ITrace BeginTrace()
 		{
-			return TraceProvider?.CurrentTrace.StartChildTrace(TraceKind.Client,
+			return TraceProvider?.CurrentTrace?.StartChildTrace(TraceKind.Client,
```

Comment:
This is not necessary; the implementation of `StartChildTrace` handles this (by design).

Reply:
Ah yes, I forgot `StartChildTrace` is an extension method.